### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by jpopelka**
I know it's not label friday yet, so you can ignore this for now

### CODE REVIEW

**Comment by jpopelka** [comment]
recheck

**Comment by jpopelka** [comment]
>     * packit-service-tests-requre : FAILURE in 18m 55s

```
E       sqlalchemy.exc.ProgrammingError: (psycopg2.ProgrammingError) relation "copr_builds" does not exist
E       LINE 1: DELETE FROM copr_builds
E                           ^
```

???

File: files/install-deps.yaml [hunk 1]
```diff
@@ -25,17 +24,15 @@
...
         state: present
     - name: Install pip deps
       pip:
         name:
-          - persistentdict
-          # temporary workaround for sake of marhsmallow
-          - git+https://github.com/packit-service/packit.git
-          - sentry-sdk==0.14.2
-          - sentry-sdk[flask]==0.14.2
+          - persistentdict # still needed by Alembic
```
Comment (lachmanfrantisek) [inline files/install-deps.yaml:34]: You mean upgrade scripts?

Comment (jpopelka) [reply files/install-deps.yaml]: Yes, I've updated the comment.

Comment (lachmanfrantisek) [reply files/install-deps.yaml]: Thanks.

**Pull request merged**